cryptography/Cargo.toml

```diff
@@ -12,17 +12,19 @@ documentation = "https://docs.rs/commonware-cryptography"
 
 [dependencies]
 commonware-codec = { workspace = true }
-commonware-utils = { workspace = true, features = ["std"] }
+commonware-utils = { workspace = true }
 bytes = { workspace = true }
 thiserror = { workspace = true }
-rand = { workspace = true, features = ["std_rng"] }
+rand = { workspace = true }
 sha2 = { workspace = true }
 blake3 = { workspace = true }
-rayon = { workspace = true }
+rayon = { workspace = true, optional = true }
 zeroize = { workspace = true, features = ["zeroize_derive"] }
-ed25519-consensus = "2.1.0"
-blst = { workspace = true, features = ["no-threads"] }
```

Comment:
We should retain this `no-threads` feature

Reply:
didn't mean to move this, restored.